File: FRONT/src/app/models/venda.ts
import { formaPag } from './formaPag';
import { ItemVenda } from './item-venda';
export interface Venda {
    id?: number;
    cliente: string;
    item?: ItemVenda;
    itemId: number;
    criadoEm?: Date;
    formapagId: number;
    formapag?: formaPag;
}

File: API/Models/Venda.cs
using System;
using System.Collections.Generic;

namespace API.Models
{
    public class Venda
    {
        public Venda() => CriadoEm = DateTime.Now;
        public int Id { get; set; }
        public string Cliente { get; set; }
        public List<ItemVenda> Itens { get; set; }

        public FormaPag FormaPag { get; set; }
        public int FormaPagId { get; set; }
        public DateTime CriadoEm { get; set; }
    }
}